dataland-keycloak/dataland_theme/email/html/email-verification.ftl

```diff
@@ -0,0 +1,18 @@
+
+<#import "template.ftl" as layout>
+<@layout.emailLayout>
+    <p>Someone has created a Dataland account with this email address. If this was you, click the link below to verify your email address</p>
+    <table role="presentation" style="border-collapse:separate;line-height:100%;">
+        <tr>
+            <td  role="presentation" style="border:none;border-radius:6px;cursor:auto;padding:11px 20px;background:#e67f3f;">
+                <a href="${msg(link)}" style="background:#e67f3f;color:#ffffff;font-family:Helvetica, sans-serif;font-weight:600;line-height:120%;Margin:0;text-decoration:none;text-transform:none;" target="_blank">
+                    Activate your account!
+                </a>
+            </td>
+        </tr>
+    </table>
+    <p><strong>This link will expire within  ${msg(linkExpirationFormatter(linkExpiration))}.</strong></p>
```

Comment:
Here the link is given as text, simply because the design from Leo and Varenya told us to do it.

But for the verification mail, we do not have a design by Leo and Varenya.
That's why we designed the verification mail with no clear goal/picture.

But it's inconsistent, you are right.   I can also add the link as text in the verification mail.

Reply:
Stephan thought that maybe a new ticket for this "frontend" task might make sense if it is too much work now.

Reply:
I have added a link given as text to the verification mail, so that it is consistent now.